src/tokenizer.cc

```diff
@@ -74,6 +74,86 @@ std::unique_ptr<Token> Tokenizer::NextToken() {
     return MakeUnique<Token>(TokenType::kEOL);
   }
 
+  if (data_[current_position_] == '"') {
+    current_position_++;  // Skip opening quote
+    std::string tok_str;
+    bool escape = false;
+    for (; current_position_ < data_.length(); current_position_++) {
+      auto c = data_[current_position_];
+      switch (c) {
+        case '\\':
+          if (!escape) {
+            escape = true;
+            continue;
+          }
```

Comment:
Does this handle a quoted \ correctly? If I give the input string of "\\n" this will get output as "\n" won't it? The first \ will mark as as escaped, the second \ will be output directly without escaping?

Reply:
I suspect github may have mangled your text - that or I'm just being thick.

If the question was `Does this handle a quoted \\ correctly?` I believe so. `\\` is the escape sequence for writing a backslash. So `\\n` is `\n` (no escape).